src/pages/token-metadata/print.md

```diff
@@ -20,8 +20,8 @@ Every Non-Fungible asset — i.e. `NonFungible` and `ProgrammableNonFungible` to
 Whenever a new printed edition is created from a printable NFT, a few things happen:
 
 - A brand new edition NFT is created and its data matches the original NFT. The only difference is that the printed edition uses a different token standard than the original NFT.
-  - For a `NonFungible` asset, the printed editions use the `NonFungibleEdition` token standard.
-  - For a `ProgrammableNonFungible` asset, the printed editions use the `ProgrammableNonFungibleEdition` token standard.
+  - For a `NonFungible` Nft, the printed editions use the `NonFungibleEdition` token standard.
+  - For a `ProgrammableNonFungible` Nft, the printed editions use the `ProgrammableNonFungibleEdition` token standard.
```

Comment:
ProgrammableNonFungible Nft also seems kind of redundant, maybe this should still be called asset here?